next/components/forms/steps/StepperView.tsx

```diff
@@ -0,0 +1,30 @@
+import { StepData } from '../types/TransformedFormData'
+import StepperViewRow from './StepperViewRow'
+
+interface StepperViewProps {
+  steps: StepData[]
+  currentStep: number
+  onChangeStep?: (stepIndex: number) => void
+}
+
+const StepperView = ({ steps, currentStep, onChangeStep }: StepperViewProps) => {
+  return (
+    <div className="p-3">
+      {steps.map((step: StepData, key: number) => (
```

Comment:
key is defined as optional string in TransformedFormData and as number here - the way order is used below it probably won't work nicely when undefined or arbitrary string is passed in

Reply:
That is different property key which has nothing to do with key from map() function. But I chose wrong naming convention because as I see, its confusing. Thanks!